test/WebJobs.Script.Tests/Description/DotNet/FunctionAssemblyLoadContextTests.cs

```diff
@@ -37,31 +39,21 @@ public void RuntimeAssemblies_AreLoadedInDefaultContext(string assemblyName)
             Assert.Same(AssemblyLoadContext.Default, AssemblyLoadContext.GetLoadContext(assembly));
         }
 
-        // TODO: FACAVAL
-        //[Fact]
-        //public void ResolveAssembly_WithIndirectPrivateDependency_LogsIfResolutionFails()
-        //{
-        //    TestLogger testLogger = new TestLogger("Test");
-
-        //    var resolver = new FunctionAssemblyLoader("c:\\");
-
-        //    var metadata1Directory = @"c:\testroot\test1";
-        //    var metadata1 = new FunctionMetadata { Name = "Test1", ScriptFile = $@"{metadata1Directory}\test.tst" };
-        //    var metadata2 = new FunctionMetadata { Name = "Test2", ScriptFile = @"c:\testroot\test2\test.tst" };
+        [Fact]
+        public void InitializeDeps_LoadsExpectedDependencies()
+        {
+            string depsPath = Path.Combine(Directory.GetCurrentDirectory(), @"Description\DotNet\TestFiles\DepsFiles");
```

Comment:
Will this work on a linux test build?

Reply:
Good catch. Had copied the path from another test and didn't modify the code. I've updated that test as part of this changes as well.